.infra/workers.ts

```diff
@@ -130,4 +130,12 @@ export const workers: Worker[] = [
     subscription: 'api-cdc',
     args: { enableMessageOrdering: true },
   },
+  {
+    topic: 'api.v1.new-notification',
+    subscription: 'api.unread-notification-count',
+  },
+  {
+    topic: 'api.v1.new-notification',
+    subscription: 'api.new-notification-redis',
```

Comment:
Question around naming, should we opt for the redis part here?
We removed it from the above one as it's technically also a redis subscription on one side.

Reply:
I followed the existing subscriptions that we have. I can change it for something else if you want.
`api.new-notification-real-time`, or `api.new-notification-subscriptions`, feel free to suggest